validation.go

```diff
@@ -118,14 +118,14 @@ func (v *subSchema) validateRecursive(currentSubSchema *subSchema, currentNode i
 
 			value := currentNode.(json.Number)
 
-			_, isValidInt64, _ := checkJsonNumber(value)
+			_, isInt := new(big.Int).SetString(string(value), 10)
```

Comment:
In the pr https://github.com/xeipuuv/gojsonschema/pull/100/files#diff-7d1c2a3334601b6c1958aae0a594cba8R80 parsing of integer in a scientific notation was added to the library, so by doing in a such way the scientific notation validation will no longer be supported and possibly will break someone's current validation

https://play.golang.org/p/RCoaFO2K3If

Reply:
@johandorland updated the regarding the dropping the support of int32 and int64 https://github.com/xeipuuv/gojsonschema/pull/193